src/EditorFeatures/VisualBasicTest/ImplementInterface/ImplementInterfaceCommandHandlerTests.vb

```diff
@@ -70,6 +70,36 @@ End Interface")
         End Function
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)>
+        Public Async Function TestInterfaceWithTuple() As Task
+            Dim code = <text>
+Imports System
+
+Class Foo
+    Implements IFoo$$
+End Class
+Interface IFoo
+    Function Method(x As (Alice As Integer, Bob As Integer)) As (String, String)
+End Interface</text>
+
+            Dim expectedText = <text>
```

Comment:
These should be in ImplementInterfaceTests.vb.  ImplementINterfaceCommandHandlerTests is for actually testing hte command handling infrastructure.